### PR TITLE
Replace construction of FileInputStream and FileOutputStream objects with Files NIO APIs.

### PR DESCRIPTION
**Pull request by arturobernalg**

### CODE REVIEW

File: src/main/java/org/apache/commons/compress/archivers/zip/ZipSplitOutputStream.java [hunk 1]
```diff
@@ -65,7 +65,7 @@ public ZipSplitOutputStream(final File zipFile, final long splitSize) throws Ill
         this.zipFile = zipFile;
         this.splitSize = splitSize;
 
-        this.outputStream = new FileOutputStream(zipFile);
+        this.outputStream = Files.newOutputStream((zipFile.toPath()));
```
Comment (bodewig) [inline src/main/java/org/apache/commons/compress/archivers/zip/ZipSplitOutputStream.java:68]: please remove the redundant pair of parenthesis

**Review by bodewig — CHANGES_REQUESTED** [review]
Many thanks @arturobernalg - there's a minor nit I'd like you to address before we can merge this.

**Comment by PeterAlfredLee** [comment]
We have some similar occurrences in tests(e.g. `TarArchiveOutputStreamTest.testCount`).

Maybe we can also replace them?

File: src/main/java/org/apache/commons/compress/archivers/zip/ZipSplitOutputStream.java (hunk 1 above)
Comment (arturobernalg) [reply src/main/java/org/apache/commons/compress/archivers/zip/ZipSplitOutputStream.java]: Changed

**Comment by arturobernalg** [comment]
> Many thanks @arturobernalg - there's a minor nit I'd like you to address before we can merge this.

Hi @bodewig,
What changes do you want me to make?

**Comment by arturobernalg** [comment]
> We have some similar occurrences in tests(e.g. `TarArchiveOutputStreamTest.testCount`).
> 
> Maybe we can also replace them?

Yep. Changed 
TY

File: src/main/java/org/apache/commons/compress/archivers/zip/ZipSplitOutputStream.java (hunk 1 above)
Comment (bodewig) [reply src/main/java/org/apache/commons/compress/archivers/zip/ZipSplitOutputStream.java]: thank you

**Comment by bodewig** [comment]
> What changes do you want me to make?

The one you already made in response to my review comment.

Thanks again.

**Pull request merged**